packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -283,6 +298,19 @@ - (NSView *)nextViewToFocusForHorizontalNavigation:(FocusZoneAction)action
 
 - (NSView *)nextViewToFocusWithFallback:(FocusZoneAction)action
 {
+
+	// Special case if we're currently focused on self
+	NSView *firstResponder = GetFirstResponder([self window]);
+	if (self == firstResponder) {
+		if (action == FocusZoneActionDownArrow)
```

Comment:
nit: what kind of tabbing or spacing is it happening?

Reply:
Like the text formatting? It's all tabs in this file.

Reply:
Justnoticed braces were off. Fixed.